src/sass/_variables.scss

```diff
@@ -12,3 +12,6 @@ $form-border-color: #292929;
 
 // section-blog-posts
 $post-info-text-color: #676767; // converted from rgb(103,103,103)
+
+//section-brands
+$gray: #cccccc;
```

Comment:
We shouldn't use variable names that tell us the value. I think `brand-color` name would be more appropriate.

Reply:
of course. i'll fix it asap :)